### PR TITLE
Add ZHA best practices for avoiding pairing difficulties

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Add ZHA best practices for avoiding pairing difficulties when adding devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Hedda** [comment]
Anyone want to review this?

**Review by Adminiuga — DISMISSED** [review]
Lgtm

**Review by ludeeus — CHANGES_REQUESTED** [review]
Please don't mix spaces and tabs, use space before and after list items

**Comment by Hedda** [comment]
> Please don't mix spaces and tabs, use space before and after list items

Done, I hope

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
- If possible try to pair your Zigbee devices in their intended final location, (and not pair it next to the Zigbee coordinator and then need to move it after). 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
  - Pairing a Zigbee device next to the Zigbee coordinator and then moving it later can result in dropped/lost connections or other issues. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
- Zigbee devices that in the past have been connected to another Zigbee coordinator (hub/bridge/gateway/controller) can usually only be added/paired in ZHA after they have been restored to their factory reset settings. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
    - If the device you want to add is not brand new and as such never paired before then you always have to make sure to first manually reset the device to its factory default settings before you will be able to add/pair it. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
- Some battery-operated Zigbee devices are known to have problems with pairing if they have Low battery voltage. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
    - Some people have reported replacing the battery on their newly received Xiaomi/Aqara devices solved pairing issues. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
- Check that you have enough Zigbee router devices (also known as Zigbee signal repeaters or range extenders) and if you do not have any, invest and add some mains-powered devices that will work as Zigbee routers. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
    - Aim to start out with mains-powered devices before adding battery-operated devices as a "weak" Zigbee network mesh (e.g., the device is too far from the Zigbee coordinator or a Zigbee router) may prevent some devices from being paired. Zigbee router devices are also needed to increase the maximum of devices that can be connected to your Zigbee mesh network. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
    - Note that some Zigbee devices are not fully compatible with all brands of Zigbee router devices. Xiaomi/Aqara devices are for example known not to work with Zigbee router devices from Centralite, General Electrics, Iris, Ledvance/OSRAM/ LIGHTIFY/Sylvania, Orvibo, PEQ, Securifi, and SmartThings/Samsung. Better results can usually be achieved by using mains-powered devices IKEA and Nue/3A Home or dedicated DIY routing devices based on Texas Instruments CC253x/CC26x2 and XBee Series 2/3 Zigbee radios. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
- Be patient as the pairing of some Zigbee devices may require multiple attempts and you may sometimes need to try again and again. 
```

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
    - Some devices, like example those from Xiaomi/Aqara, are known to not be 100% compliant with the standard Zigbee specifications and may therefore require many paring attempts over 10-20 minutes or longer. 
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
### Best practices for avoiding pairing difficulties
```

titles should be the same or +/- as the previous title

Is it correct that this should be under "Adding devices"?
Is not under troubleshooting a better place?

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown]:
```suggestion
```

https://deploy-preview-17466--home-assistant-docs.netlify.app/integrations/zha/#add-philips-hue-bulbs-that-have-previously-been-added-to-another-bridge

This is already documented on the same page.

File: source/_integrations/zha.markdown
Comment (ludeeus) [inline source/_integrations/zha.markdown:319]: Is it correct that this should be under "Adding devices"? Is under troubleshooting not a better place for this?

Comment (Hedda) [reply source/_integrations/zha.markdown:319]: I think that this should be under the "Adding devices" as these are things you always need to consider when adding Zigbee devices, but feel free to move it if you think that it fits better there. You can always move it later once this is merged.

Comment (ludeeus) [reply source/_integrations/zha.markdown:319]: I'll leave that decision to the codeowners

Comment (Adminiuga) [reply source/_integrations/zha.markdown:319]: I think it makes more sense to have it under "adding devices" section.
Be, should the numbering be updated? Both are  1?

Comment (Hedda) [reply source/_integrations/zha.markdown:319]: Those numberings are not for the new section in this PR but for the old section above it. Regardless, the numberings in that othering section look correct when looking at the whole file.

**Review by ludeeus — APPROVED** [review]

**Pull request merged**
